### PR TITLE
Validate certificate via root CA

### PR DESCRIPTION
**Pull request by pho-enix**

### CODE REVIEW

File: octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp [hunk 1]
```diff
@@ -147,6 +167,15 @@ void Octopus::initBno055() {
   }
 }
 
+void Octopus::initBno280() {
```
Comment (Yannic92) [inline octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp]: I guess this should be initBme280?

File: octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp [hunk 2]
```diff
@@ -147,6 +167,15 @@ void Octopus::initBno055() {
...
+  Printer::printMsg("Octopus", "Initializing BNO280: ");
```
Comment (Yannic92) [inline octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp]: Here again BME280

**Review by Yannic92 — CHANGES_REQUESTED** [review]
Looks good to me.
Tested it and it works. 
Just the two naming issues should be fixed.

Thank you for this PR! 👍

**Comment by Yannic92** [comment]
Targets Issue #13

**Comment by pho-enix** [comment]
Naming issues resolved. Furthermore resolved the "Sensor reading failure" issue.

**Review by Yannic92 — APPROVED** [review]

File: octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp [hunk 3]
```diff
@@ -118,34 +120,52 @@ const Bno055Values Octopus::readBno055() {
...
 void Octopus::initBme680() {
   Printer::printMsg("Octopus", "Initializing BME680: ");
-  if (this->bme680.begin(118)) {
+  if (this->bme680.begin(0x76)) {
     this->bme680.setTemperatureOversampling(BME680_OS_8X);
     this->bme680.setHumidityOversampling(BME680_OS_2X);
     this->bme680.setPressureOversampling(BME680_OS_4X);
     this->bme680.setIIRFilterSize(BME680_FILTER_SIZE_3);
     this->bme680.setGasHeater(320, 150); // 320*C for 150 ms
+    bme280Ready = true;
```
Comment (Yannic92) [inline octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp]: I think this should be bme680Ready?

File: octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp [hunk 4]
```diff
@@ -118,34 +120,52 @@ const Bno055Values Octopus::readBno055() {
...
     Serial.println("OK");
   } else {
+    bme280Ready = false;
```
Comment (Yannic92) [inline octopus-hub-things-example/src/main/octopus/octopus-mqtt/octopus.cpp]: I think this should be bme680Ready?

**Review by Yannic92 — CHANGES_REQUESTED** [review]

**Review by Yannic92 — APPROVED** [review]

**Pull request merged**